sources/mysql/infoschema.go

```diff
@@ -219,20 +219,51 @@ func (isi InfoSchemaImpl) GetColumns(conv *internal.Conv, table common.SchemaAnd
 		} else {
 			colAutoGen = ddl.AutoGenCol{}
 		}
+		defaultVal := schema.DefaultValue{
+			IsPresent: colDefault.Valid,
+			Value:     "",
+		}
+		if colDefault.Valid {
+			defaultVal.Value = sanitizeDefaultValue(colDefault.String)
+		}
 		c := schema.Column{
-			Id:      colId,
-			Name:    colName,
-			Type:    toType(dataType, columnType, charMaxLen, numericPrecision, numericScale),
-			NotNull: common.ToNotNull(conv, isNullable),
-			Ignored: ignored,
-			AutoGen: colAutoGen,
+			Id:           colId,
+			Name:         colName,
+			Type:         toType(dataType, columnType, charMaxLen, numericPrecision, numericScale),
+			NotNull:      common.ToNotNull(conv, isNullable),
+			Ignored:      ignored,
+			AutoGen:      colAutoGen,
+			DefaultValue: defaultVal,
 		}
 		colDefs[colId] = c
 		colIds = append(colIds, colId)
 	}
 	return colDefs, colIds, nil
 }
 
+// sanitizeDefaultValue removes extra characters added to Default Value.
+// These extra characters are charset which are not supported by spanner.
+// Example_1:
+// Default Value: "concat('John', 'swamp', 'span')"
+// Default Value with extra characters: "concat(_utf8mb4\\'John\\',_utf8mb4\\'swamp\\',_utf8mb4\\'span\\')"
+// Default Value after removing "_utf8mb4": "concat( \\'John\\',  \\'swamp\\',  \\'span\\')"
+// Default Value after removing all backslashes: "concat('John','swamp','span')"
+// Example_2:
+// Default Value: "This is a message \\\\nwith a newline\\\\rand a carriage return."
```

Comment:
`Default Value: "This is a message \\\\nwith a newline\\\\rand a carriage return."` -> `Default Value: 'This is a message \nwith a newline\rand a carriage return.'`